### PR TITLE
[update_documentation] Updated comments for setVideoUrl

### PR DESCRIPTION
**Pull request by mityaha**

### CODE REVIEW

File: js/google_ima.js [hunk 1]
```diff
@@ -1972,6 +1972,8 @@ require("../html5-common/js/utils/utils.js");
      * @public
      * @method GoogleIMAVideoWrapper#setVideoUrl
      * @param {string} url The new url to insert into the video element's src attribute
+     * @param {string} encoding The encoding of video stream, possible values are found in OO.VIDEO.ENCODIN (unused here)
```
Comment (aldigun) [inline js/google_ima.js]: type ENCODING

File: js/pulse.js [hunk 2]
```diff
@@ -859,6 +859,8 @@
          * @public
          * @method PulseVideoWrapper#setVideoUrl
          * @param {string} url The new url to insert into the video element's src attribute
+         * @param {string} encoding The encoding of video stream, possible values are found in OO.VIDEO.ENCODIN (unused here)
```
Comment (aldigun) [inline js/pulse.js]: typo ENCODING

File: js/google_ima.js (hunk 1 above)
Comment (mityaha) [inline js/google_ima.js]: done

File: js/pulse.js (hunk 2 above)
Comment (mityaha) [inline js/pulse.js]: done

**Pull request merged**